### PR TITLE
disable Unbekannt folder in web.de

### LINKED ISSUE #56: add nice texts for gmx.de and web.de
i think, both need imap enabled.

for web.de, we should also tell the user to switch of his "inbox sorting". not sure what exactly this was, i think we need to create a test account and try out.

### PR DESCRIPTION
**Pull request by missytake**
closes #56 

disabling the "Ungelesen" folder doesn't seem to be necessary: https://hilfe.web.de/email/empfangen-und-lesen/ungelesen.html#.pc_page.freemail.erste-schritte.teaser_5.hilfe
it leaves a copy in the Inbox.

only the "Unbekannt" folder is deadly to contact requests.

WIP because not all details regarding GMX seem to be sorted out - even if the page already looks good enough imho.

### CODE REVIEW

**Comment by Simon-Laux** [comment]
gmx and web.de ARE NOT the SAME THING. Even though they belong to eachother.

File: _providers/web.de.md [hunk 1]
```diff
@@ -47,10 +47,23 @@ server:
     username_pattern: emaillocalpart
 before_login_hint: |
   You must allow IMAP access to your account before you can login.
```
Comment (Simon-Laux) [inline _providers/web.de.md]: After login hint about disabling the 'Unbekannt' folder is missing here.

**Comment by Simon-Laux** [comment]
the preview is somehow broken

**Comment by missytake** [comment]
broken preview: see https://github.com/deltachat/provider-db/pull/70

File: _providers/web.de.md [hunk 2]
```diff
@@ -47,6 +47,10 @@ server:
...
+after_login_hint: |
+  Note: if you have your web.de spam settings too strict, you won't receive contact requests from new people.
+  If you want to receive contact requests, you should disable the "3-Wege-Spamschutz" in the web.de settings. 
+  Read how: https://hilfe.web.de/email/spam-und-viren/spamschutz-einstellungen.html
```
Comment (Simon-Laux) [inline _providers/web.de.md]: Maybe those new users won't know what contact requests are? So I'm not sure on the wording here.
Besides that the hint is good IMO.

Comment (missytake) [reply _providers/web.de.md]: Might be that they don't know how exactly they work in the Delta context - but in general they probably understand that they won't see messages from new people, maybe even better than if we say "you won't see messages from new people".

Do you have a different proposal? This was the best I could think of...

**Comment by Simon-Laux** [comment]
So web.de is always in german so the login hint can be german too, I guess.
```german
Wenn nachrichte von neuen Kontakten (aka. Kontanktanfragen) fehlen,
stellen Sie bitte sicher das der "3-Wege-Spamschutz" in Ihren web.de-Einstellungen DEAKTIVIERT ist.
Anleitung: https://hilfe.web.de/email/spam-und-viren/spamschutz-einstellungen.html
```
Something like that?

**Pull request merged**